OpenOversight/app/main/views.py

```diff
@@ -543,26 +544,20 @@ def submit_complaint():
 @main.route('/submit', methods=['GET', 'POST'])
 @limiter.limit('5/minute')
 def submit_data():
+    preferred_dept_id = 1
```

Comment:
The only hiccup here is if primary key = 1 doesn't exist (since this ID is from an autoincrementing primary key, this can happen if the current primary department is not the first one that was created, i.e. if a department was made during testing and deleted). We can instead do something like:

```
preferred_dept_id = Department.query.first().id
```

to be more robust